<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1436</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="start_play">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Старт</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_names">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>271</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Список городов</string>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>781</width>
     <height>491</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ready_to_answer">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>570</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Я знаю</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>490</y>
     <width>471</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Поле для ввода название города</string>
   </property>
  </widget>
  <widget class="QLabel" name="win_message">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>700</y>
     <width>1091</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(2, 79, 0, 255), stop:0.19 rgba(50, 147, 22, 255), stop:0.275 rgba(236, 191, 49, 255), stop:0.39 rgba(243, 61, 34, 255), stop:0.555 rgba(135, 81, 60, 255), stop:0.667 rgba(121, 75, 255, 255), stop:0.825 rgba(164, 255, 244, 255), stop:0.885 rgba(104, 222, 71, 255), stop:1 rgba(93, 128, 0, 255))</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="choice_city">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="asnwer_city">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>560</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="move_list">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>120</y>
     <width>151</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>------&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
